continuous_integration/environment-3.8.yaml

```diff
@@ -25,7 +25,7 @@ dependencies:
   # sqlalchemy 1.4.0 causes deprecation warnings to be raised from pandas
   # along with other issues https://github.com/pandas-dev/pandas/issues/40467
   - sqlalchemy<1.4.0
-  - pyarrow=1.0
+  - pyarrow=4.0
```

Comment:
Since we already pin to `1.0` above in the 3.7 case, changed this to `4.0`. It is currently also latest, but it would be good to keep testing if `5.0` comes around in the future